test/server/ip-tools.js

```diff
@@ -33,6 +33,20 @@ describe("IP tools", () => {
 		assert.deepEqual(cidrRange, stringRange);
 	});
 
+	it('should not parse invalid ranges', () => {
+		assert.equal(IPTools.stringToRange('42.42.10.0 - 42.42.5.0'), null);
+		assert.equal(IPTools.stringToRange('250.0.0.0 - 260.0.0.0'), null);
```

Comment:
```suggestion
		assert.equal(IPTools.stringToRange('250.0.0.0 - 260.0.0.0'), null);
		assert.equal(IPTools.stringToRange('250.0.0.0/43'), null);
		assert.equal(IPTools.stringToRange('250.0.0.0/16x'), null);
		assert.equal(IPTools.stringToRange('250.0.0.0.1/24'), null);
		assert.equal(IPTools.stringToRange('250.0.0.0.1 - 250.0.0.2'), null);
```

This PR might not be able to validate all of these, but it probably should.

Reply:
Sure.